### PR TITLE
detach inverse in the VAMPE score

### PR DESCRIPTION
**Pull request by DanielWicz**
In the original paper, it is stated:

"Another advantage of the VAMP-E based validation score is that it does not involve any inverse
operation of matrices and can be stably computed."

It seems in general, that the `sym_inverse` should be detached from the graph, otherwise it is involved in the computational graph and gradient is calculated through the inverse. In fact, from my point of view, both inverses and `koopman` should be detached from the graph, and only the `torch.trace` part should be contributing to the optimization.

### CODE REVIEW

**Comment by clonker** [comment]
Hi! Thank you very much for your contributions :) please correct me if I'm wrong but I think we'd still need the gradient flow through these components - while the score doesn't directly involve any inverses, we still have to transform our data into whitened space (C^{-1/2}). Have you tried/compared training with your modification to the reference?

Thanks!

**Comment by DanielWicz** [comment]
At the moment I have tested in my own implementation in Tensorflow 2. I have noticed that it lead to faster training and faster reduction of the loss. Therefore I have decided to open this Pull request. I will try to make more controlled tests with deeptime. 

Do you use some particular dataset for comparison, for example like WLALL pentapeptide from PYEMMA ?

**Review by JamesonDiamond1 — APPROVED** [review]
@JamesonDiamond1
